hr_timesheet_activity_begin_end/models/account_analytic_line.py

```diff
@@ -1,23 +1,7 @@
 # -*- coding: utf-8 -*-
-#
-#
-#    Authors: Guewen Baconnier
-#    Copyright 2015 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# © 2015 Camptocamp SA (author: Guewen Baconnier)
+# © 2017 Martronic SA
```

Comment:
According to https://github.com/OCA/maintainer-tools/pull/197 we should use `Copyright`.

Non-blocking comment

Reply:
💯

Reply:
@alexey-pelykh I fully agree witih you. But somehow I find more interesting to have this merged than to block further this old PR.
Thanks for your review.

Reply:
I say that I'd agree, yet not agree enough to hit merge, and I'd opt in for second PSC member opinion